src/skmultiflow/drift_detection/kswin.py

```diff
@@ -89,7 +91,7 @@ def __init__(self, alpha=0.005, window_size=100, stat_size=30, data=None):
         if self.window_size < self.stat_size:
             raise ValueError("stat_size must be smaller than window_size")
 
-        if type(data) != np.ndarray or type(data) is None:
+        if not isinstance(data, np.ndarray) or isinstance(data, None):
```

Comment:
```suggestion
        if not isinstance(data, np.ndarray) or data is None:
```

This could be the culprit making the tests fail.

Reply:
You are right!